app/Repositories/HTTP/HTTPRepository.php

```diff
@@ -158,4 +158,9 @@ public function search($key, $data)
         return $search_data;
```

Comment:
You are to modify the returned data and not just doing it on your own
The resourch is to modify the returned data in the controller and not here